packages/account-v2/src/mocks/idv-form.mock.ts

```diff
@@ -0,0 +1,124 @@
+import { ResidenceList } from '@deriv/api-types';
+
+export const DOCUMENT_LIST = [
```

Comment:
Follow one naming convention
Make a standard
Either use camelCase or PascalCase or snake_case

Reply:
Content of mock folder is just to test components functionality. It will be removed. But I think the convention for using configs is Upper Snake case if I'm  not wrong